odoo/addons/base/models/res_lang.py

```diff
@@ -334,6 +334,23 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def action_activate_langs(self):
```

Comment:
Not sure, but it seems we add an action in base that is overridden in website by a wizard, also defined in base ?